silkworm/silkrpc/core/remote_state.cpp

```diff
@@ -54,28 +54,28 @@ Task<uint64_t> AsyncRemoteState::previous_incarnation(const evmc::address& /*add
     co_return 0;
 }
 
-Task<std::optional<silkworm::BlockHeader>> AsyncRemoteState::read_header(uint64_t block_number, const evmc::bytes32& block_hash) const noexcept {
+Task<std::optional<silkworm::BlockHeader>> AsyncRemoteState::read_header(BlockNum block_number, const evmc::bytes32& block_hash) const noexcept {
```

Comment:
The same change here and in the rest of functions in the module should be applied also in `remote_state.hpp`.

Reply:
ok